pkg/mover/intermediate.go

```diff
@@ -94,7 +94,11 @@ func tarChart(tfw *tarFileWriter, chart *chart.Chart) error {
 }
 
 func packImages(tfw *tarFileWriter, imageChanges []*internal.ImageChange, logger Logger) error {
-	imagesTarFilename, err := tarImages(imageChanges, logger)
+	cacheDir := cacheDir()
+	if err := os.MkdirAll(cacheDir, 0750); err != nil {
```

Comment:
why mkdirAll? wouldn't be safer to use mkdir? 

re: permissions, do we need the group to have access to it? wouldn't 700 be enough?

Reply:
ok

Reply:
```suggestion
	if err := os.Mkdir(cacheDir, 0700); err != nil {
```